packages/components/src/product-icon/README.md

```diff
@@ -0,0 +1,45 @@
+ProductIcon Component
```

Comment:
I think it would be useful that if in this README we have a step-by-step reference of how to add a new icon if we want to.